shardy/dialect/sdy/ir/attrs.td

```diff
@@ -66,13 +66,13 @@ def Sdy_Mesh : AttrDef<Sdy_Dialect, "Mesh"> {
 
   Here are some examples of meshes:
 
-    - An empty mesh represents a placeholder mesh that can be replaced during
-      propagation: <[]>
-    - A mesh with an unnamed axis and an explicit device ID, which is typically
-      used to represent maximal sharding: <[], device_ids=[3]>
-    - A mesh with two axes and implicit device IDs iota(6): <["a"=2, "b"=3]>
-    - A mesh with two axes and explicit device IDs specifying the device
-      ordering: <["a"=3, "b"=2], device_ids=[0, 2, 4, 1, 3, 5]>
+  - An empty mesh represents a placeholder mesh that can be replaced during
+    propagation: <[]>
+  - A mesh with an unnamed axis and an explicit device ID, which is typically
+    used to represent maximal sharding: <[], device_ids=[3]>
+  - A mesh with two axes and implicit device IDs iota(6): <["a"=2, "b"=3]>
+  - A mesh with two axes and explicit device IDs specifying the device
+    ordering: <["a"=3, "b"=2], device_ids=[0, 2, 4, 1, 3, 5]>
```

Comment:
MeshAttr is missing constraints, you can get them from this method: https://github.com/openxla/shardy/blob/main/shardy/dialect/sdy/ir/verifiers.cc#L648

Reply:
Please let me know if I missed anything.